Watchman.Discord/Areas/BotSettings/Controllers/VersionController.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Devscord.DiscordFramework.Framework.Architecture.Controllers;
+using Devscord.DiscordFramework.Framework.Architecture.Middlewares;
+using Devscord.DiscordFramework.Middlewares.Contexts;
+using Devscord.DiscordFramework.Services;
+using Watchman.Cqrs;
+
+namespace Watchman.Discord.Areas.BotSettings.Controllers
+{
+    public class VersionController : IController
+    {
+        private readonly IQueryBus queryBus;
+        private readonly ICommandBus commandBus;
+
+        public VersionController(IQueryBus queryBus, ICommandBus commandBus)
+        {
+            this.queryBus = queryBus;
+            this.commandBus = commandBus;
+        }
+
+        [DiscordCommand("-version")]
+        public void PrintVersion(string message, Dictionary<string, IDiscordContext> contexts)
+        {
+            var version = File.ReadAllText("version.txt");
```

Comment:
czytanie zawartości pliku w kontrolerze to zła praktyka

to powinno być coś wbudowane w framework
lub jakieś query, którego handler pobiera dane z niewiadomego (dla kontrolera) miejsca
(to może być plik, baza, jakieś api)

zrób query-handler który będzie pobierał numer wersji